### PR TITLE
Add `process.path` and `process.args`

### LINKED ISSUE #7: Add new properties for the path and arguments separately
Currently there's `process.cmd`, but it includes both the path and arguments. Would be useful for https://github.com/sindresorhus/fkill-cli/issues/16 to have separate `process.path` and `process.args` properties.

Context:

> I looked into this more, and I might have a solution to get the arguments without the full path. It seems `ps -o pid -o comm` returns the full path without arguments and `ps -o pid -o command` returns with arguments. So we could get both columns and just diff them to get the arguments. I think this should be added as a new property in [`ps-list`](https://github.com/sindresorhus/ps-list). So we could add an `args` property with just the arguments and a `path` property with the full path to the binary without arguments, just for completeness.

### PR DESCRIPTION
**Pull request by yaodingyd**
fix #7

### CODE REVIEW

**Comment by sindresorhus** [comment]
Can you confirm this follows the description in https://github.com/sindresorhus/ps-list/issues/7:

> I looked into this more, and I might have a solution to get the arguments without the full path. It seems ps -o pid -o comm returns the full path without arguments and ps -o pid -o command returns with arguments. So we could get both columns and just diff them to get the arguments. I think this should be added as a new property in ps-list. So we could add an args property with just the arguments and a path property with the full path to the binary without arguments, just for completeness.

**Comment by sindresorhus** [comment]
This is missing docs and TS definitions.

**Comment by yaodingyd** [comment]
Yes, my implementation follows the suggestion: 
1. use `comm` for `path` is pretty straightforward
2. `args` is alias for `command`. I use `value.args.split(value.comm)[1].trim()` to achieve diff between `comm` and `args` results.

**Comment by sindresorhus** [comment]
// @stroncium

File: index.js [hunk 1]
```diff
@@ -51,6 +51,8 @@ const main = async (options = {}) => {
 			pid: Number.parseInt(key, 10),
 			name: path.basename(value.comm),
 			cmd: value.args,
+			path: value.comm,
+			args: value.args.split(value.comm)[1].trim(),
```
Comment (sindresorhus) [inline index.js]: I would prefer using `startsWith` and `slice` here. The current way will fail if the `args` contain an argument that is the same as `comm`.

Comment (yaodingyd) [reply index.js]: I don't get you. 1) `comm` only returns the path name; 2) if they are the same, that means there is no argument and it returns empty string

**Comment by stroncium** [comment]
It doesn't have conflicts with master in terms of code, but does have them in terms of functionality atm.

**Comment by yaodingyd** [comment]
@stroncium Could you elaborate?

**Comment by stroncium** [comment]
@yaodingyd https://github.com/sindresorhus/ps-list/commit/285f434b1114823a5cc1cec710ab8253ca36cebc

Tests fail after merge. And sine the changes are only to the case of multiple `ps` runs(which is left of for the moment in case parsing single `ps` run output fails, so won't even be triggered normally).

**Comment by yaodingyd** [comment]
I can wait till 30 is finished.

**Pull request closed without merging**